lib/stat_tracker.rb

```diff
@@ -462,6 +462,78 @@ def biggest_team_blowout(team_id)
       diff = (game.home_goals - game.away_goals).abs
       biggest_blowout = diff if diff > biggest_blowout
     end
-    biggest_blowout
+    biggest_blowout  end
```

Comment:
Is this formatting right or is github being weird?

Reply:
I didn't touch that so probably resolving a merge conflict did it, updated